<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>354</width>
                <height>604</height>
            </rect>
        </property>
        <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="windowTitle">
            <string>InstaHelper</string>
        </property>
        <property name="windowIcon">
            <iconset>
                <normaloff>../src/InstaHelper.png</normaloff>../src/InstaHelper.png
            </iconset>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QTabWidget" name="tabWidget">
                <property name="geometry">
                    <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>361</width>
                        <height>581</height>
                    </rect>
                </property>
                <property name="tabPosition">
                    <enum>QTabWidget::South</enum>
                </property>
                <property name="tabShape">
                    <enum>QTabWidget::Triangular</enum>
                </property>
                <property name="currentIndex">
                    <number>0</number>
                </property>
                <property name="usesScrollButtons">
                    <bool>true</bool>
                </property>
                <property name="documentMode">
                    <bool>false</bool>
                </property>
                <property name="tabsClosable">
                    <bool>false</bool>
                </property>
                <property name="tabBarAutoHide">
                    <bool>true</bool>
                </property>
                <widget class="QWidget" name="homeTab">
                    <attribute name="title">
                        <string>Home | Follow</string>
                    </attribute>
                    <widget class="QLabel" name="profilePicLabel">
                        <property name="geometry">
                            <rect>
                                <x>250</x>
                                <y>20</y>
                                <width>91</width>
                                <height>91</height>
                            </rect>
                        </property>
                        <property name="frameShape">
                            <enum>QFrame::StyledPanel</enum>
                        </property>
                        <property name="frameShadow">
                            <enum>QFrame::Sunken</enum>
                        </property>
                        <property name="text">
                            <string/>
                        </property>
                        <property name="pixmap">
                            <pixmap>../../../../../.designer/src/profile_picture.jpg</pixmap>
                        </property>
                        <property name="scaledContents">
                            <bool>true</bool>
                        </property>
                        <property name="wordWrap">
                            <bool>false</bool>
                        </property>
                    </widget>
                    <widget class="QLabel" name="label_2">
                        <property name="geometry">
                            <rect>
                                <x>10</x>
                                <y>10</y>
                                <width>71</width>
                                <height>21</height>
                            </rect>
                        </property>
                        <property name="font">
                            <font>
                                <weight>75</weight>
                                <bold>true</bold>
                            </font>
                        </property>
                        <property name="text">
                            <string>Username</string>
                        </property>
                    </widget>
                    <widget class="QLabel" name="usernameLabel">
                        <property name="geometry">
                            <rect>
                                <x>90</x>
                                <y>10</y>
                                <width>141</width>
                                <height>21</height>
                            </rect>
                        </property>
                        <property name="text">
                            <string/>
                        </property>
                    </widget>
                    <widget class="QProgressBar" name="progressBar">
                        <property name="geometry">
                            <rect>
                                <x>10</x>
                                <y>510</y>
                                <width>331</width>
                                <height>31</height>
                            </rect>
                        </property>
                        <property name="value">
                            <number>0</number>
                        </property>
                        <property name="invertedAppearance">
                            <bool>false</bool>
                        </property>
                    </widget>
                    <widget class="QTableWidget" name="suggestedTableWidget">
                        <property name="geometry">
                            <rect>
                                <x>10</x>
                                <y>150</y>
                                <width>331</width>
                                <height>321</height>
                            </rect>
                        </property>
                        <property name="autoFillBackground">
                            <bool>false</bool>
                        </property>
                        <property name="frameShape">
                            <enum>QFrame::NoFrame</enum>
                        </property>
                        <property name="frameShadow">
                            <enum>QFrame::Raised</enum>
                        </property>
                        <property name="lineWidth">
                            <number>1</number>
                        </property>
                        <property name="sizeAdjustPolicy">
                            <enum>QAbstractScrollArea::AdjustIgnored</enum>
                        </property>
                        <property name="autoScroll">
                            <bool>false</bool>
                        </property>
                        <property name="autoScrollMargin">
                            <number>5</number>
                        </property>
                        <property name="editTriggers">
                            <set>QAbstractItemView::NoEditTriggers</set>
                        </property>
                        <property name="dragDropOverwriteMode">
                            <bool>false</bool>
                        </property>
                        <property name="alternatingRowColors">
                            <bool>false</bool>
                        </property>
                        <property name="selectionMode">
                            <enum>QAbstractItemView::ExtendedSelection</enum>
                        </property>
                        <property name="selectionBehavior">
                            <enum>QAbstractItemView::SelectRows</enum>
                        </property>
                        <property name="textElideMode">
                            <enum>Qt::ElideRight</enum>
                        </property>
                        <property name="showGrid">
                            <bool>false</bool>
                        </property>
                        <property name="gridStyle">
                            <enum>Qt::SolidLine</enum>
                        </property>
                        <property name="cornerButtonEnabled">
                            <bool>true</bool>
                        </property>
                        <property name="rowCount">
                            <number>0</number>
                        </property>
                        <property name="columnCount">
                            <number>5</number>
                        </property>
                        <attribute name="horizontalHeaderVisible">
                            <bool>false</bool>
                        </attribute>
                        <attribute name="horizontalHeaderCascadingSectionResizes">
                            <bool>false</bool>
                        </attribute>
                        <attribute name="horizontalHeaderMinimumSectionSize">
                            <number>50</number>
                        </attribute>
                        <attribute name="horizontalHeaderDefaultSectionSize">
                            <number>50</number>
                        </attribute>
                        <attribute name="horizontalHeaderHighlightSections">
                            <bool>false</bool>
                        </attribute>
                        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                            <bool>false</bool>
                        </attribute>
                        <attribute name="horizontalHeaderStretchLastSection">
                            <bool>false</bool>
                        </attribute>
                        <attribute name="verticalHeaderVisible">
                            <bool>false</bool>
                        </attribute>
                        <attribute name="verticalHeaderMinimumSectionSize">
                            <number>50</number>
                        </attribute>
                        <attribute name="verticalHeaderDefaultSectionSize">
                            <number>50</number>
                        </attribute>
                        <attribute name="verticalHeaderHighlightSections">
                            <bool>false</bool>
                        </attribute>
                        <column>
                            <property name="text">
                                <string>profile</string>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>Name</string>
                            </property>
                            <property name="font">
                                <font>
                                    <weight>75</weight>
                                    <bold>true</bold>
                                </font>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>follower</string>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>follow</string>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>id</string>
                            </property>
                        </column>
                    </widget>
                    <widget class="QLabel" name="label_3">
                        <property name="geometry">
                            <rect>
                                <x>10</x>
                                <y>40</y>
                                <width>71</width>
                                <height>21</height>
                            </rect>
                        </property>
                        <property name="font">
                            <font>
                                <weight>75</weight>
                                <bold>true</bold>
                            </font>
                        </property>
                        <property name="text">
                            <string>Full Name</string>
                        </property>
                    </widget>
                    <widget class="QLabel" name="label_4">
                        <property name="geometry">
                            <rect>
                                <x>10</x>
                                <y>70</y>
                                <width>71</width>
                                <height>21</height>
                            </rect>
                        </property>
                        <property name="font">
                            <font>
                                <weight>75</weight>
                                <bold>true</bold>
                            </font>
                        </property>
                        <property name="text">
                            <string>Followers</string>
                        </property>
                    </widget>
                    <widget class="QLabel" name="label_5">
                        <property name="geometry">
                            <rect>
                                <x>10</x>
                                <y>100</y>
                                <width>71</width>
                                <height>21</height>
                            </rect>
                        </property>
                        <property name="font">
                            <font>
                                <weight>75</weight>
                                <bold>true</bold>
                            </font>
                        </property>
                        <property name="text">
                            <string>Following</string>
                        </property>
                    </widget>
                    <widget class="QLabel" name="fullNameLabel">
                        <property name="geometry">
                            <rect>
                                <x>90</x>
                                <y>40</y>
                                <width>141</width>
                                <height>21</height>
                            </rect>
                        </property>
                        <property name="text">
                            <string/>
                        </property>
                    </widget>
                    <widget class="QLabel" name="followersLabel">
                        <property name="geometry">
                            <rect>
                                <x>90</x>
                                <y>70</y>
                                <width>151</width>
                                <height>21</height>
                            </rect>
                        </property>
                        <property name="text">
                            <string/>
                        </property>
                    </widget>
                    <widget class="QLabel" name="followingLabel">
                        <property name="geometry">
                            <rect>
                                <x>90</x>
                                <y>100</y>
                                <width>151</width>
                                <height>21</height>
                            </rect>
                        </property>
                        <property name="text">
                            <string/>
                        </property>
                    </widget>
                    <widget class="Line" name="line">
                        <property name="geometry">
                            <rect>
                                <x>10</x>
                                <y>120</y>
                                <width>331</width>
                                <height>20</height>
                            </rect>
                        </property>
                        <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                        </property>
                    </widget>
                    <widget class="Line" name="line_2">
                        <property name="geometry">
                            <rect>
                                <x>220</x>
                                <y>10</y>
                                <width>20</width>
                                <height>111</height>
                            </rect>
                        </property>
                        <property name="orientation">
                            <enum>Qt::Vertical</enum>
                        </property>
                    </widget>
                    <widget class="QPushButton" name="showMoreButton">
                        <property name="geometry">
                            <rect>
                                <x>130</x>
                                <y>480</y>
                                <width>89</width>
                                <height>25</height>
                            </rect>
                        </property>
                        <property name="cursor">
                            <cursorShape>PointingHandCursor</cursorShape>
                        </property>
                        <property name="tabletTracking">
                            <bool>false</bool>
                        </property>
                        <property name="text">
                            <string>Show More</string>
                        </property>
                    </widget>
                </widget>
                <widget class="QWidget" name="unfollowTab">
                    <attribute name="title">
                        <string>Unfollow</string>
                    </attribute>
                </widget>
                <widget class="QWidget" name="Subscription">
                    <attribute name="title">
                        <string>Subscription</string>
                    </attribute>
                    <widget class="QPushButton" name="subscriptionActivateButton">
                        <property name="geometry">
                            <rect>
                                <x>250</x>
                                <y>40</y>
                                <width>89</width>
                                <height>25</height>
                            </rect>
                        </property>
                        <property name="cursor">
                            <cursorShape>PointingHandCursor</cursorShape>
                        </property>
                        <property name="text">
                            <string>Actiavate</string>
                        </property>
                    </widget>
                    <widget class="QLabel" name="label">
                        <property name="geometry">
                            <rect>
                                <x>10</x>
                                <y>20</y>
                                <width>121</width>
                                <height>17</height>
                            </rect>
                        </property>
                        <property name="text">
                            <string>Subscription Key</string>
                        </property>
                    </widget>
                    <widget class="QLineEdit" name="subscriptionKeyEdit">
                        <property name="geometry">
                            <rect>
                                <x>10</x>
                                <y>40</y>
                                <width>211</width>
                                <height>25</height>
                            </rect>
                        </property>
                        <property name="echoMode">
                            <enum>QLineEdit::PasswordEchoOnEdit</enum>
                        </property>
                    </widget>
                    <widget class="QGroupBox" name="groupBox">
                        <property name="enabled">
                            <bool>false</bool>
                        </property>
                        <property name="geometry">
                            <rect>
                                <x>10</x>
                                <y>110</y>
                                <width>331</width>
                                <height>421</height>
                            </rect>
                        </property>
                        <property name="title">
                            <string>Automatic Tasks</string>
                        </property>
                        <widget class="QLabel" name="label_7">
                            <property name="geometry">
                                <rect>
                                    <x>10</x>
                                    <y>59</y>
                                    <width>161</width>
                                    <height>31</height>
                                </rect>
                            </property>
                            <property name="text">
                                <string>Auto Unfollow</string>
                            </property>
                        </widget>
                        <widget class="QCheckBox" name="checkBox">
                            <property name="geometry">
                                <rect>
                                    <x>250</x>
                                    <y>60</y>
                                    <width>71</width>
                                    <height>31</height>
                                </rect>
                            </property>
                            <property name="cursor">
                                <cursorShape>PointingHandCursor</cursorShape>
                            </property>
                            <property name="text">
                                <string>Enable</string>
                            </property>
                            <property name="tristate">
                                <bool>false</bool>
                            </property>
                        </widget>
                        <widget class="QDialogButtonBox" name="buttonBox">
                            <property name="geometry">
                                <rect>
                                    <x>110</x>
                                    <y>390</y>
                                    <width>81</width>
                                    <height>25</height>
                                </rect>
                            </property>
                            <property name="cursor">
                                <cursorShape>PointingHandCursor</cursorShape>
                            </property>
                            <property name="standardButtons">
                                <set>QDialogButtonBox::Apply</set>
                            </property>
                        </widget>
                        <widget class="QLabel" name="label_8">
                            <property name="geometry">
                                <rect>
                                    <x>10</x>
                                    <y>126</y>
                                    <width>151</width>
                                    <height>21</height>
                                </rect>
                            </property>
                            <property name="text">
                                <string>User's follower limit</string>
                            </property>
                        </widget>
                        <widget class="QSpinBox" name="spinBox_2">
                            <property name="geometry">
                                <rect>
                                    <x>160</x>
                                    <y>120</y>
                                    <width>81</width>
                                    <height>31</height>
                                </rect>
                            </property>
                            <property name="wrapping">
                                <bool>false</bool>
                            </property>
                            <property name="specialValueText">
                                <string>0</string>
                            </property>
                            <property name="accelerated">
                                <bool>false</bool>
                            </property>
                            <property name="suffix">
                                <string/>
                            </property>
                            <property name="minimum">
                                <number>999</number>
                            </property>
                            <property name="maximum">
                                <number>1000000</number>
                            </property>
                            <property name="singleStep">
                                <number>1000</number>
                            </property>
                            <property name="value">
                                <number>5000</number>
                            </property>
                        </widget>
                        <widget class="QCheckBox" name="checkBox_2">
                            <property name="geometry">
                                <rect>
                                    <x>250</x>
                                    <y>120</y>
                                    <width>71</width>
                                    <height>31</height>
                                </rect>
                            </property>
                            <property name="cursor">
                                <cursorShape>PointingHandCursor</cursorShape>
                            </property>
                            <property name="text">
                                <string>Enable</string>
                            </property>
                            <property name="tristate">
                                <bool>false</bool>
                            </property>
                        </widget>
                    </widget>
                    <widget class="QLabel" name="subscriptionStatusLabel">
                        <property name="geometry">
                            <rect>
                                <x>10</x>
                                <y>70</y>
                                <width>211</width>
                                <height>17</height>
                            </rect>
                        </property>
                        <property name="text">
                            <string>You are not activated</string>
                        </property>
                    </widget>
                </widget>
            </widget>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
